### PR TITLE
Telegram payments guide

### PR DESCRIPTION
**Pull request by LWJerri**
Progress can be tracked here: https://github.com/grammyjs/website/issues/155

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-  "recommendations": [
```
Comment (KnorpelSenf) [inline .vscode/extensions.json:2]: Please revert

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,19 +1,19 @@
 {
-  "deno.config": "./deno.json",
```
Comment (KnorpelSenf) [inline .vscode/settings.json:2]: Same

File: deno.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
-  "fmt": {
```
Comment (KnorpelSenf) [inline deno.json:2]: Same

File: site/docs/guide/payments.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+---
+prev: ./deployment-types.md
```
Comment (KnorpelSenf) [inline site/docs/guide/payments.md:2]: Awesome changes. Please write the rest.

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

File: site/docs/guide/payments.md (hunk 4 above)
Comment (LWJerri) [reply site/docs/guide/payments.md:2]: > Awesome changes. Please write the rest.

Yeap, xD) I'm passing the last exams tomorrow and plan to write some text from my notes.

**Review by roj1512 — CHANGES_REQUESTED** [review]
Revert the unrelated changes.

**Comment by LWJerri** [comment]
@KnorpelSenf, can you please take a quick look, is this the kind of information presentation you expect or not?

File: site/docs/guide/deployment-types.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 prev: ./games.md
+next: ./telegram-payments.md
```
Comment (KnorpelSenf) [inline site/docs/guide/deployment-types.md:3]: I'd rather have the payments page after games but before deployment types. Deploying is what happens last.

File: site/docs/guide/telegram-payments.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+::: warning Compatibility with devices
+To access **Payments 2.0**, users must have **Telegram 7.7** or higher. For the desktop version, it must be **2.7.2** or higher.
+
+Outdated _mobile_ apps released after May 2017 only support basic chat payments with bots.
+:::
+
+## Prerequisites
+
+To get started and understand how payments work in Telegram, you need to choose a payment provider from [this](https://core.telegram.org/bots/payments#supported-payment-providers) list.
```
Comment (Nazar-Ant) [inline site/docs/guide/telegram-payments.md:19]: The focus should be on the list

```suggestion
To get started and understand how payments work in Telegram, you need to choose a payment provider from this [list](https://core.telegram.org/bots/payments#supported-payment-providers).
```

File: site/docs/guide/telegram-payments.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+**Payment Provider** - this is a company that offers online services for merchants and banks to facilitate electronic payments, including smart cards, bank payments like direct debits, and other banking transactions.
+
+Choosing a payment provider can be challenging as they often require detailed information about you and your project, and they charge a percentage of each payment based on that. Telegram, as mentioned before, does not charge any commission for payments.
```
Comment (Nazar-Ant) [inline site/docs/guide/telegram-payments.md:23]:
```suggestion
Choosing a payment provider can be challenging as they often require detailed information about you and your project, and they charge a percentage of each payment based on that.
Telegram, as mentioned before, does not charge any commission for payments.
```

**Comment by KnorpelSenf** [comment]
@LWJerri is this still a thing?

**Comment by LWJerri** [comment]
> @LWJerri is this still a thing?

uuh, mhh... yes? But I don't know when I finish it, uuh...

**Comment by LWJerri** [comment]
> @LWJerri is this still a thing?

The structure for this guide is already in issue, I'm just too damn lazy to do anything...

**Comment by roj1512** [comment]
Closing due to inactivity.

**Pull request closed without merging**